advanced/src/server/index.js

```diff
@@ -0,0 +1,75 @@
+import 'isomorphic-fetch'
+
+import React from 'react';
+import {StaticRouter, matchPath} from 'react-router-dom';
+import express from 'express';
+import {renderToString} from 'react-dom/server';
+import {
+    TipserElementsProvider,
+    SsrTipserElementsProvider,
+    ComponentsStateSsrManager
+} from '@tipser/tipser-elements';
+
+import App from '../client/App';
+
+const assets = require(process.env.RAZZLE_ASSETS_MANIFEST);
+
+const server = express();
+
+const POS_ID = '59e86b79b8f3f60a94ecd26a';
+
+server
+    .disable('x-powered-by')
+    .use(express.static(process.env.RAZZLE_PUBLIC_DIR))
+    .get('/*', (req, res) => {
+        const componentsStateSsrManager = new ComponentsStateSsrManager(POS_ID, 'prod')
+        const context = {};
+
+        const toRender = (
+            <TipserElementsProvider posId={POS_ID}>
+                <SsrTipserElementsProvider componentsStateManager={componentsStateSsrManager}>
+                    <StaticRouter context={context} location={req.url}>
+                        <App/>
+                    </StaticRouter>
+                </SsrTipserElementsProvider>
+            </TipserElementsProvider>
+        );
+
+        renderToString(toRender);
```

Comment:
Please add a short comment here explaining why we do this renderToString here ("dry run" that is used to figure out ids in use)

Reply:
done